### PR TITLE
Character card component

### PR DESCRIPTION
**Pull request by JivanaLipsewers**
hard coded as placeholder

### CODE REVIEW

File: components/card/card.js [hunk 1]
```diff
@@ -1 +1,35 @@
-export function createCharacterCard() {}
+export function createCharacterCard() {
+    const characterImageSrc = "https://rickandmortyapi.com/api/character/avatar/1.jpeg";
+    const characterName = "Rick Sanchez";
+    const characterStatus = "Alive";
+    const characterType = null;
+    const characterOccurrences = 51;
```
Comment (vaupunkt) [inline components/card/card.js]: you want to have these as arguments for your function, so you can create Chards for random characters - yours only works for Rick. If you use the function later you want to fill in the different arguments with the content from the fetched data.
```suggestion
export function createCharacterCard(characterImageSrc, characterName, characterStatus, characterType, characterOccurrences) {
```

File: components/card/card.js [hunk 2]
```diff
@@ -1 +1,35 @@
...
+    </div>`
+  const newCard = document.createElement("li");
+  newCard.classList.add("card");
+  newCard.innerHTML = cardContent;
+
+  return newCard
+
+}
+
+console.log(createCharacterCard());
```
Comment (vaupunkt) [inline components/card/card.js]: you don't need this anymore :)

**Review by vaupunkt — CHANGES_REQUESTED** [review]

File: components/card/card.js (hunk 1 above)
Comment (josch87) [reply components/card/card.js]: This was hardcoded on purpose! We want to make it dynamic in part2!

Comment (makitango) [reply components/card/card.js]: We would like to keep it as an object which is why we left it hard-coded for this section

**Pull request merged**